README

```diff
@@ -469,7 +464,7 @@ To get a list of the internal commands type "HELP" at the prompt.
 In addition, the following commands are available:
 
 MOUNT "Emulated Drive letter" "Real Drive or Directory"
-      [-t type] [-noioctl] [-usecd number] [-size drivesize]
+      [-t type] [-usecd number] [-size drivesize]
```

Comment:
![2019-12-15_20-17](https://user-images.githubusercontent.com/1557255/70878856-02b25680-1f78-11ea-9f66-0f6577d37f66.png)
Knock out `-usecd` and `-cd` references.